docs/sources/project/get-help.md

```diff
@@ -2,103 +2,127 @@ page_title: Where to chat or get help
 page_description: Describes Docker's communication channels
 page_keywords: IRC, Google group, Twitter, blog, Stackoverflow
 
+<style>
+/* @TODO add 'no-zebra' table-style to the docs-base stylesheet */
+/* Table without "zebra" striping */
+.content-body table.no-zebra tr {
+  background-color: transparent;
+}
+</style>
+
 # Where to chat or get help
 
-There are several communications channels you can use to chat with Docker community members and developers. 
+There are several communications channels you can use to chat with Docker
+community members and developers.
 
-<style type="text/css">
-.tg  {border-collapse:collapse;border-spacing:0;text-align: left;}
-.tg td{font-family:Arial, sans-serif;font-size:14px;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;vertical-align:top;}
-.tg th{font-family:Arial, sans-serif;font-size:14px;font-weight:normal;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;}
-</style>
-<table class="tg">
+<table>
```

Comment:
Yep. I've noted all the table "areas" but that's just in case.  I like your table styles and I'd love to use them. So, if they are on master and we pull the changes from this PR --- complete with your excise of my <style> tags --- we should get a preview of this content with your styling.  If not we'll tweak it till we get it all in a line and I'll cherry-pick/push the whole lot live.